### ISSUE TITLE
error when trying to install the package via jsdelivr

### ISSUE THREAD

**Issue by barbieswimcrew** [kimyvgy/simple-scrollspy#58]
Hi, I am using [jsdelivr](https://www.jsdelivr.com/) to install the package. This has worked perfectly so far and ran up to version 2.4.2 without any problems. With version 2.5.0, however, the package can no longer be installed in this way. 

Please have a look: 

2.4.2 works: https://cdn.jsdelivr.net/npm/simple-scrollspy@2.4.2/+esm
2.5.0 doesn't: https://cdn.jsdelivr.net/npm/simple-scrollspy@2.5.0/+esm

The following error appears:

```
/**
 * Failed to bundle using Rollup v2.79.1: failed to resolve an internal import.
 * If you believe this to be an issue with jsDelivr, and not with the package itself, please open an issue at https://github.com/jsdelivr/jsdelivr
 */
```

Any clue on how to get that work? Thanks in advance!

**Comment by kimyvgy** [comment]
@barbieswimcrew This issue is fixed in the commit 44a192e.

![image](https://github.com/user-attachments/assets/3ba78fee-c797-4864-bc18-ae1c2f4550d1)

**Comment by barbieswimcrew** [comment]
Hi @kimyvgy thanks for your work! jsDelivr seems to work properly now! :-)

Unfortunately there's a new problem since v2.5.3... if I import scrollSpy as usual:

```js
import scrollSpy from 'simple-scrollspy';

scrollSpy(this.element, {...some options...})
```

there is a new error now:

`TypeError: scrollSpy is not a function`

I don't know how to solve this... tried to do something like `new scrollSpy()` but then it says

`TypeError: scrollSpy is not a constructor`

Any hint, what I might do wrong here? Thanks in advance!
